Gemfile

```diff
@@ -28,6 +28,7 @@ group :test do
   gem 'grape-entity', '~> 0.6'
   gem 'maruku'
   gem 'mime-types'
+  gem 'multi_xml'
```

Comment:
We have integration tests with this, and we don't want to load multi_xml for all tests, why was it required?

Reply:
How does that work in practice? When I checkout the code in master, run bundle install, then run rspec, I get these test failures ...

```
  1) ActiveSupport::XmlMini uses multi_xml
     Failure/Error: expect(Grape::Xml).to eq(::MultiXml)
     
     NameError:
       uninitialized constant MultiXml
       Did you mean?  MultiJson
     # ./spec/integration/multi_xml/xml_spec.rb:7:in `block (2 levels) in <top (required)>'
```

Reply:
Nevermind, I re-read the [contributing document](https://github.com/ruby-grape/grape/blob/master/CONTRIBUTING.md) and noticed I was running the tests incorrectly. I ran things through rake and it works without it.

I remove the gem dependency here 18c0310. Sorry about that.

Reply:
No worries. It would be nice if we could make it so `rspec` runs successfully by default, since that's a common pattern.